### PR TITLE
Update app.js to enable usage with django-content-editor

### PR DESCRIPTION
**Pull request by frague59**
Add a try/catch block in the `resolveElementArray` function to avoid issues on CKEditor initialization. See https://github.com/feincms/django-content-editor/issues/30#issuecomment-2379439783

Fixes https://github.com/hvlads/django-ckeditor-5/issues/253

### CODE REVIEW

**Comment by frague59** [comment]
I'm not sure which branch of the ternary operator I should use.

**Comment by goapunk** [comment]
@frague59 thanks :) according to the linked issue the problem comes from passing the wrong parameter in the callback for the formset event, wouldn't it be enough to change line 113 `django.jQuery(document).on("formset:added", createEditors);` to ` django.jQuery(document).on("formset:added", () => createEditors());` then ? :) Could you test if that's already enough to make it work ?

**Comment by frague59** [comment]
Thanks for your quick answer.

I've tried the proposed patch about `django.jQuery(document).on("formset:added", () => createEditors());`, but  I've still an issue about the `element.matches(query)` not defined in the browser, at the CKEditor creation. Editor is created, but the other handlers bound to the `formset:added` event are failing.

File: django_ckeditor_5/static/django_ckeditor_5/app.js [hunk 1]
```diff
@@ -26,22 +26,27 @@ function getCookie(name) {
...
+ *
  * @returns {array.<HTMLElement>}
  */
 function resolveElementArray(element, query) {
-    return element.matches(query) ? [element] : [...element.querySelectorAll(query)];
+    try {
+        return element.matches(query) ? [element] : [...element.querySelectorAll(query)];
+    } catch (err) {
+        console.warn(err)
+        return [element]
```
Comment (goapunk) [inline django_ckeditor_5/static/django_ckeditor_5/app.js:40]: from what I've understand we should probably return an empty array here if no element was found

**Comment by goapunk** [comment]
> Thanks for your quick answer.
> 
> I've tried the proposed patch about `django.jQuery(document).on("formset:added", () => createEditors());`, but I've still an issue about the `element.matches(query)` not defined in the browser, at the CKEditor creation. Editor is created, but the other handlers bound to the `formset:added` event are failing.

@frague59 thanks for testing. Interesting that it doesn't fix it. Could you test a again and simply remove that line completely? I'm not actually sure that's the cause of the issue, it might be the `MutationObserver`. Is your project public so we can test ourselves?

**Pull request closed without merging**